src/js/edu-benefits/5490/config/form.js

```diff
@@ -498,11 +502,25 @@ const formConfig = {
                   'ui:options': {
                     hideIf: form => {
                       const status = _.get('highSchool.status', form);
-                      return status !== 'graduated';
+                      return status !== 'graduated' && status !== 'graduationExpected';
                     },
-                    expandUnder: 'status'
-                  }
-                }
+                    expandUnder: 'status',
```

Comment:
Not sure I understand what the issue is here? Aren't we currently checking if `expandUnder` is a string? What about it now isn't working for what we're trying to do?